_includes/provider-ocp-add-container.md

```diff
@@ -10,18 +10,7 @@
 5.  Enter the appropriate **Zone** for the provider. If you do not
     specify a zone, it is set to `default`.
 
-6.  From the **Alerts** list, select **Prometheus** to enable external
-    alerts. Selecting **Prometheus** adds an **Alerts** tab to the lower
-    pane to configure the Prometheus service. Alerts are disabled by
-    default.
-
-7.  From the **Metrics** list, select **Hawkular** or **Prometheus** to
-    collect capacity and utilization data, or leave as **Disabled**.
-    Selecting **Prometheus** or **Hawkular** adds a **Metrics** tab to
-    the lower pane for further configuration. Metrics are disabled by
-    default.
```

Comment:
These drop-down no longer exist, they have been moved to endpoint tabs with the default type being "Disabled"